dapp/src/solc.rs

```diff
@@ -75,7 +75,7 @@ impl<'a> SolcBuilder<'a> {
             .join(",");
 
         // tracing::trace!(?lib_paths);
-        solc = solc.args(["--allow-paths", &lib_paths]);
+        solc = solc.args(std::array::IntoIter::new(["--allow-paths", &lib_paths]));
```

Comment:
Why is this change needed?

Reply:
The iterator impl that you were using didn't exist in 1.51